packages/components/src/FormatFile/InternalThumbnail.tsx

```diff
@@ -61,3 +58,12 @@ function getIconNameFromType(mimeType: string): IconNames {
       return "file";
   }
 }
+
+function isSupportedImageType(file: FileUpload) {
```

Comment:
Tyler and I chatted and we are ok with the duplication of this method since this is a temporary fix until the library that we use ([sharp](https://sharp.pixelplumbing.com)) will add this support and if we happen to duplicate this again before that, we will then abstract it